postgres.tf

```diff
@@ -75,6 +75,12 @@ data "azurerm_subnet" "ss_subnet_pre_postgresql" {
   resource_group_name  = data.azurerm_resource_group.ss_resource_group.name
 }
 
+// Resource Group (this is the resource group where the privatelink.postgres.database.azure.com resides)
+data "azurerm_resource_group" "privatelink_resource_group" {
+  name = "core-infra-intsvc-rg"
```

Comment:
remove unused `var. DNSResGroup` if going with static value

Reply:
commented this var for now